### ISSUE TITLE
Query results pane horizontal scroll disappearing

### ISSUE THREAD

**Issue by wellsoliver** [jakob/Postico#835]
On MacOS Ventura 13.2.1 and the horizontal scroll for my query results is just gone. It kind of appears and then I go to mouse over it to scroll and it goes way. Makes things a little unusable. Resizing the window makes the scroller visible for a second and if I'm lucky I can grab it.

Postico 2.0.1 (9593)

**Comment by jakob** [comment]
Huh. I've just updated to Ventura 13.2.1 and done some testing. While it does behave in the way you describe when scrolling with the trackpad, I think this is just how scroll views on macOS work now? I also did some testing with Ventura 13.0 in a VM, and with older builds of Postico, and the behaviour seemed to be the same.

Is this something that has changed recently? Does Postico behave differently than other apps on your Mac? I've just tried Numbers, and it seems to do the same thing.

What kind of mouse / trackpad are you using? If you have a mouse with a scroll wheel, you can hold down shift to scroll horizontally. If you have a trackpad, you can show the scrollers by tapping with two fingers.

You can also try changing the scroll bar settings (in System Settings -> Appearance).

(Unrelated remark: Over the years, it's been a semi-regular customer support question how to scroll horizontally in the query result pane. The invisible scrollbars on macOS are a usability nightmare.)

**Comment by wellsoliver** [comment]
Yeah, this is weird, and happening less frequently on the latest Ventura. I agree the disappearing scroll thing is a general usability nightmare. I think we're OK closing this and I will pop in here again if I see it happening a lot more.

**Comment by wellsoliver** [comment]
This is happening to me quite a bit again. Ventura 13.31, Postico 2.0.1 (9593) - it just disappears, and resizing very very briefly returns, then I go to catch it to scroll my results horizontally, and it's gone.

**Comment by jakob** [comment]
Another customer has reported this issue. I'm not 100% sure what is going on. If it happens again, can you et me know

1) What kind of mouse you have

2) What your scrollbar setting in System Settings is (see screenshot)
![Image 06 06 23 at 13 04](https://github.com/jakob/Postico/assets/198305/f91e24db-e29a-4147-a59f-e2267e649bf7)

As a workaround, you can try this:

1) Hold down shift while scrolling with the scroll wheel to scroll horizontally

2) Change the scroll bar settings to "always"

**Comment by jlannoy** [comment]
Hello.

As said via email first, I have the same trouble with Postico 1.5.22 and OS X 13.4.
My mouse is Logitech M280 and the settings are "Automatically based on mouse".
No other apps had the problem, restarting the computer did make it work again.

I'll check for more other apps when it happens again, and I'll check your workarounds.

**Comment by wellsoliver** [comment]
So I work on three computers: an older iMac, a 2018 MBP, and a 2022 Mac Studio. Same version of MacOS, same version of Postico on each, never an issue on the first two, but it never works on the Mac Studio. I can't scroll. Mouse on the Mac Studio is bluetooth-connected Logitech MX Master, show scroll bars set to "Automatically..."

Setting scroll to always works, and is a much better experience imho so I'm good.

**Comment by jakob** [comment]
OK, so the issue seems to happen with Logitech mice. I also have a Logitech MX Master on my iMac, but I'm not seeing the issue. Maybe it's because I uninstalled the Logitech software because it didn't work with Universal Control.

@jlannoy @wellsoliver do you have Logi Options installed?

**Comment by wellsoliver** [comment]
well, fwiw, same Logitech mouse on my 2019 iMac and no issues with the scrollbar. I do not have Logi Options installed.

**Comment by jlannoy** [comment]
Well... I installed Logi Options recently on the mac where the problem happens. It may be related. I'll uninstall it and we will see. From my pov you can close the issue too.
